docs/getting_started.rst

```diff
@@ -333,6 +333,31 @@ teracy-dev Git Clone and Vagrant Up
          "VBoxDrv.inf" in your installation directory then re-install it to fix the issue. The VirtualBox
          has an installation issue which was reported `here <https://www.virtualbox.org/ticket/4140>`_
 
+       - On Windows, if you see the error `code converter not found (UTF-16LE to Windows-1258) (Encoding::ConverterNotFoundError)` when using vagrant:
+
+        ::
+
+          $ vagrant status
+          [TeracyDev::Util][WARN]: The _id: '0' is deprecated, use the _id: 'kernel-core' instead for {"_id"=>"0", "location"=>{"git"=>{"remote"=>{"origin"=>"https://github.com/teracyhq-incubator/teracy-dev-core.git"}, "branch"=>"develop"}}, "require_version"=>">= 0.4.0-SNAPSHOT"}
+
+          [TeracyDev::Plugin][INFO]: installing plugin: {"_id"=>"essential-0", "name"=>"vagrant-hostmanager", "version"=>"", "state"=>"installed", "config_key"=>"hostmanager", "enabled"=>true, "options"=>{"enabled"=>true, "manage_host"=>true, "manage_guest"=>true}, "sources"=>["https://rubygems.org/", "https://gems.hashicorp.com/"]}
```

Comment:
we should keep relevant message, remove `The _id: '0' is deprecated ...`, remove this line `installing plugin:...`